.travis.yml

```diff
@@ -165,15 +165,21 @@ scripts:
 
 matrix:
   include:
+    - <<: *opam
+      os: linux
+      env: COMPILER=4.12.0+trunk REPOSITORIES=--repo=default,beta=git://github.com/ocaml/ocaml-beta-repository.git
```

Comment:
I suggest using the 4.12 alpha release at this point.

Reply:
I would argue against. In my experience running `+trunk` is more stable than running any `~alpha`/`~beta`. The changes are slow and rather incremental. If something is done in the `4.12` branch and break something here then it should be reported / fixed. But again, in my experience, once branched out, the alpha/beta branches are quite stable and contain bug fixes that are not in the `~alpha`/`~beta` releases.

If that can help you I could add it to the `allow_failures` list though.

Reply:
I agree with your argument, thanks. I think it does belong in `allow_failures` for how the Lwt CI is set up.